InsiteTeamTask/Repositories/DataRepository.cs

```diff
@@ -16,25 +16,39 @@ public List<Attendance> GetAttendanceListFor(int gameNumber)
             List<Member> members = service.Members().ToList();
             List<Ticket> tickets = service.Tickets().ToList();
             List<Product> products = service.Products().ToList();
+            List<Game> games = service.Games().ToList();
+            List<Season> seasons = service.Seasons().ToList();
 
             List<Attendance> attendanceList = new List<Attendance>();
 
-            for(int i = 0; i < members.Count; i++)
-            {
-                attendanceList.Add(new Attendance()
-                {
-                    Barcode = "N/A",
-                    MemberId = members[i].Id
-                });
-            }
+            //for (int i = 0; i < members.Count; i++)
+            //{
+            //    attendanceList.Add(new Attendance()
+            //    {
+            //        Barcode = "N/A",
+            //        MemberId = members[i].Id
+            //    });
+            //}
+
+            //for (int i = 0; i < tickets.Count; i++)
+            //{
+            //    attendanceList.Add(new Attendance()
+            //    {
+            //        Barcode = tickets[i].Barcode,
+            //        MemberId = 0
+            //    });
+            //}
 
-            for (int i = 0; i < tickets.Count; i++)
+            for (int i = 0; i < products.Count; i++)
             {
-                attendanceList.Add(new Attendance()
+                if (products[i].GameId == 3)
```

Comment:
What happens if we pass any value other than 3 into this method? We'd expect to retrieve products which correspond to that game number, so the check needed here is
```
if (products[i] == gameId)
```

Furthermore, I'd recommend that you have a look at LINQ for manipulating data in C# (there's a [tutorial here](https://csharp.net-tutorials.com/linq/introduction/)). This would significantly reduce the amount of code you'd have to write to perform the same looping operation:
```
matchingProducts = products.Where(p => p.GameId == gameId);
```